.github/workflows/run-nethermind-tests.yml

```diff
@@ -201,6 +201,42 @@ jobs:
       run: |
         dotnet test -c Release src/Nethermind/Nethermind.State.Test
 
+  neth-tests6:
+    name: Running Nethermind Tests 6
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Setting up dotnet
+      uses: actions/setup-dotnet@v1
```

Comment:
Please update to `actions/setup-dotnet@v2`.

Reply:
Done :) Thx!